src/main/java/uk/gov/hmcts/reform/cpo/domain/CasePaymentOrder.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.cpo.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Data;
+
+import java.util.Date;
+
+@Data
+public class CasePaymentOrder {
+
+    @JsonIgnore
```

Comment:
Assume we're going to be adding an additional UUID field to this class and the entity (following some pending confirmation)? We need to pass something to clients invoking the APIs so they know what ID to pass us for updates.